dart/lib/src/profiling.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../sentry.dart';
+
+@internal
+abstract class ProfilerFactory {
```

Comment:
```suggestion
abstract class SentryProfilerFactory {
```

prefix all classes with `Sentry` to avoid naming clashes

Reply:
done